cmd/errtrace/toolexec.go

```diff
@@ -224,3 +222,13 @@ func readBuildSHA() (_ string, ok bool) {
 	}
 	return sha, sha != ""
 }
+
+// isStdLib checks if the current execution is for stdlib using
+// the package name, and arguments.
+func isStdLib(pkg string, args []string) bool {
+	if strings.Contains(pkg, ".") {
+		return false
+	}
+
+	return slicesContains(args, "-std")
```

Comment:
The "." heuristic is only valid for the import path, not the package name.

Is just `-std` not enough?

Reply:
`pkg` is actually an import path, just poorly named. That said, `-std` check should be enough, just a little slower since we have to iterate over all of `args`, but probably not worth optimizing for that at this point.